src/app/core/campaign/campaign-store.service.ts

```diff
@@ -367,13 +367,17 @@ export class CampaignStoreService {
 
   private putCampaign(): Observable<{ prevId: number; state: CampaignState }> {
     return this.campaignFirst$.pipe(
-      switchMap(state => {
-        return this.campaignService.putCampaign(state).pipe(
-          map(newState => {
-            return { prevId: state.remoteCampaign ? state.remoteCampaign.id : null, state: newState };
-          })
-        );
-      })
```

Comment:
fake the campaign put/post by just returning the campaign from the state as the new/updated campaign. So on new campaigns, the changes won't contain the new campaign id. The campaign store eventually gets the campaign id when it reloads the campaign on route change.